nucliadb/nucliadb/search/predict.py

```diff
@@ -350,14 +350,17 @@ async def query(
             logger.warning(error)
             raise SendToPredictError(error)
 
+        params = {
+            "text": sentence,
+            "rephrase": str(rephrase),
```

Comment:
this can be `None`

Reply:
I will switch it needs to be False or True and False by default